README.md

```diff
@@ -1 +1,138 @@
-# This project is being actively developed and maintained as [Opendiffy](https://github.com/opendiffy/diffy).
+Twitter has decided to shelve open source Diffy project which is built on Twitter's outdated tech stack. We are reinventing in this space internally to accommodate newer use cases of Twitter services. Based on the internal success of these project, we will evaluate if we want to open source Twitter's future solutions in this space.   
+We won't be accepting pull requests or responding to issues for this project anymore.
+
+This project's original author (and former twitter employee) maintain their own version of this project, called [Opendiffy](https://github.com/opendiffy/diffy).
```

Comment:
"This project's original author (and former twitter employee) maintain ..." =>
"Note that @puneetkhanduri, this project's original author and a former Twitter employee, maintains ..."